spec/services/wallets/validate_service_spec.rb

```diff
@@ -71,5 +71,66 @@
         expect(result.error.messages[:granted_credits]).to eq(['invalid_granted_credits'])
       end
     end
+
+    context 'with recurring transaction rules' do
+      let(:rules) do
+        [
+          {
+            rule_type: 'interval',
+            interval: 'monthly',
+          },
+          {
+            rule_type: 'threshold',
+            threshold_credits: '1.0',
+          },
+        ]
+      end
+      let(:args) do
+        {
+          customer:,
+          organization_id: organization.id,
+          paid_credits:,
+          granted_credits:,
+          recurring_transaction_rules: rules
```

Comment:
Style/TrailingCommaInHashLiteral: Put a comma after the last item of a multiline hash.